supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -128,8 +133,8 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
 
   private lazy val standardParameters = Seq(rcJesOutput)
   // PBE this shouldn't need to be defined here, could be at the actor factory level.
-  private lazy val jesAttributes = JesAttributes(backendConfig)
-  private lazy val returnCodeContents = Try(returnCodeGcsPath.toAbsolutePath.contentAsString)
+  private lazy val jesAttributes = JesAttributes(googleConfig, configurationDescriptor)
+  protected lazy val returnCodeContents = Try(returnCodeGcsPath.toAbsolutePath.contentAsString)
```

Comment:
At one point I thought I needed to override it in my testing subclass, but it looks like that's no longer true so it can be reverted to private.